### ISSUE TITLE
TypeScript Support

### ISSUE THREAD

**Issue by EisenbergEffect** [AnomalyInnovations/serverless-es7#3]
Per our other conversations in GitHub, it would be awesome to see a `serverless-typescript` plugin that works the same as this babel-based plugin 😄

**Comment by jayair** [comment]
Yeah, we are using https://github.com/elastic-coders/serverless-webpack for this repo. If you come across something similar for TS let us know!
